xml/depl_nodes.xml

```diff
@@ -5250,27 +5250,43 @@ Set to <systemitem>true</systemitem> to use Ceph deployed with Crowbar.
   </para>
   <variablelist>
    <varlistentry>
-    <term><guimenu>Logging</guimenu>: <guimenu>Verbose</guimenu></term>
+    <term><guimenu>Trustee Domain</guimenu>: <guimenu>Delegate trust to
+    cluster users if required</guimenu></term>
     <listitem>
      <para>
-      Increases the amount of information that is written to the log files when
-      set to <guimenu>true</guimenu>.
+      Deploying Kubernetes clusters in a cloud without an Internet connection
+      (as described in <xref linkend="sec.deploy.kubernetes.without"/>)
+      requires the <literal>registry_enabled</literal> option in its cluster
+      template set to <literal>true</literal>. To make this offline scenario
+      work, you also need to set the <guimenu>Delegate trust to cluster users if
+      required</guimenu> option to <literal>true</literal>. This restores the old,
```

Comment:
No need for that, as this text is in the chapter that covers the Magnum barclamp/Crowbar. :-)